docs/snapshotting/snapshot-support.md

```diff
@@ -153,7 +153,17 @@ Now that the microVM is paused, you can create a snapshot, which can be either a
 one or a `diff` one. Full snapshots always create a complete, resume-able snapshot of
 the current microVM state and memory. Diff snapshots save the current microVM state
 and the memory dirtied since the last snapshot (full or diff). Diff snapshots are not
-resume-able, but can be merged into a full snapshot using external (provided) tooling.
+resume-able, but can be merged into a full snapshot by copying layers one by one over
+the first memory snapshot, in the order in which they were created. Here is an example
```

Comment:
Having a hard time understanding this

**Layers**? => from the diff mem states? What are these layers actually? Are they referred to in another section of the doc?
**First memory snapshot** => First can be confusing here...maybe you are referring to the last full memory snapshot?
in the order in which **they** were created => who is they? the layers? the diff mem snapshots?

Reply:
I added more information which can hopefully make it clearer to the users. PTAL!